doc/Makefile

```diff
@@ -10,6 +10,8 @@ SPHINXBUILD  = sphinx-build
 PAPER        =
 BUILDDIR     = _build
 SOURCEDIR    = src
+LIVEHOST     = 127.0.0.1
```

Comment:
I guess this has no effect for the local build, right? It still works for me.

Reply:
It works locally for me as well. But on second thought, probably better to write this as `localhost` instead of the numerical version, since that's probably more what people expect to see in their browser. I'll change it to that.